### DISCUSSION TITLE
Cabling breakout cables and uplinks in the DCIM

### DISCUSSION THREAD

**Discussion by fziegert** [netbox-community/netbox#15288] [Help Wanted!]
### Change Type

Addition

### Area

Features

### Proposed Changes

Hello everyone, 
so far I have only worked with rack tables and am looking at NetBox for the first time. I am enthusiastic about Netbox, but I need two functions for my work with Netbox:
At the moment I don't know how I can or should implement them:
My first problem is:
I have uplinks to suppliers at each of my sites where I can only get one cable into the rack. I can't see a counterpart here.
So far I have marked these ports, where the uplink is plugged in, as reserved in rack tables and added a note.
How do I do this with NetBox? As I have seen, I always have to select an opposing interface. Is it possible to implement this feature?

My second problem is with the breakout cables. I only use break-out cables in the DAC area. I've already found that you can use breakout cables and I think it's great! In our cabling it is sometimes the case that not all interfaces are used by the breakout cable. Sometimes the second or third cable is free and unfortunately I don't know how to implement this so that it is clear which of the individual breakout cables is plugged into which port.
It would be great if I could leave one cable (for example 2) free or give each part of the breakout cable its own label.

Can you help me if I have overlooked any functions or would it be possible to implement something like this? Thank you very much!

**Comment by a084ed22** [comment]
> I have uplinks to suppliers at each of my sites

assuming these are connectivity providers, you likely want to register them as circuits.

**Comment by fziegert** [comment]
> assuming these are connectivity providers, you likely want to register them as circuits.
@a084ed22 Thank you very much! That definitely solves one of my problems!

**Comment by fziegert** [comment]
To solve my problem with the breakout cables, it might be possible to create child interfaces for each individual cable. Currently, however, it is only possible for a virtual interface to be a child interface. It would be great if I could select SFP28, for example.

It is important to me that I can see in the cable trace that it is actually one cable, which splits into four. And secondly, that I can see exactly how many cables I have.
